cluster/tls_transport_test.go

```diff
@@ -89,7 +99,7 @@ func TestFinalAdvertiseAddr(t *testing.T) {
 			require.Nil(t, err)
 			if tc.expectedPort == 0 {
 				require.True(t, tc.expectedPort < port)
-				require.True(t, uint32(port) <= uint32(1<<32-1))
+				require.True(t, uint16(port) <= uint16(1<<16-1))
```

Comment:
This assertion is irrelevant, since casting a var to uint16 will _by definition_ ensure that it is <= 1<<16-1.